content/docs/faq-versioning.md

```diff
@@ -46,3 +48,19 @@ In general, we *don't* bump the major version number for changes to:
 This policy is designed to be pragmatic: certainly, we don't want to cause headaches for you. If we bumped the major version for all of these changes, we would end up releasing more major versions and ultimately causing more versioning pain for the community. It would also mean that we can't make progress in improving React as fast as we'd like.
 
 That said, if we expect that a change on this list will cause broad problems in the community, we will still do our best to provide a gradual migration path.
+
+### If a Minor Release Includes No New Features, Why Isn't It a Patch? {#minors-versus-patches}
+
+It's possible that a minor release will not include new features. [This is allowed by semver](https://semver.org/#spec-item-7):
+
+> [A minor version] MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes.
```

Comment:
Is there a way I can render this blockquote without the yellow box treatment? Or should I rewrite to use an inline quote?

<img width="931" alt="Screen Shot 2019-09-24 at 4 55 17 PM" src="https://user-images.githubusercontent.com/3624098/65558600-469f3d80-deec-11e9-9385-281597b8adc4.png">

Reply:
no, all the MD blockquotes have that

Reply:
Alex is right. This is the default `<blockquote>` styling for the website.